<!DOCTYPE html>
<html>

  <head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  <title>research</title>
  <meta name="description" content="Welcome to the Hellenthal Group website! We are based in London, at UCL Genetics Institute. ">

  
  
  <link rel="stylesheet" href="http://localhost:4000/hellenthal-group/assets/style.css">

  <link rel="canonical" href="http://localhost:4000/hellenthal-group/research/">
  <link rel="alternate" type="application/rss+xml" title="hellenthal-group" href="http://localhost:4000/hellenthal-group/feed.xml">

  <script async defer src="https://buttons.github.io/buttons.js"></script>
</head>


  <body>

    <header class="border-bottom-thick px-2 clearfix">
  <div class="left sm-width-full py-1 mt-1 mt-lg-0">
    <a class="align-middle link-primary text-accent" href="/hellenthal-group/">
      hellenthal-group
    </a>
  </div>
  <div class="right sm-width-full">
    <ul class="list-reset mt-lg-1 mb-2 mb-lg-1">
      
        
      
        
        <li class="inline-block">
          <a class="align-middle link-primary mr-2 mr-lg-0 ml-lg-2" href="/hellenthal-group/group-members/">
            group members
          </a>
        </li>
        
      
        
      
        
        <li class="inline-block">
          <a class="align-middle link-primary mr-2 mr-lg-0 ml-lg-2" href="/hellenthal-group/media/">
            media
          </a>
        </li>
        
      
        
        <li class="inline-block">
          <a class="align-middle link-primary mr-2 mr-lg-0 ml-lg-2" href="/hellenthal-group/openings/">
            openings
          </a>
        </li>
        
      
        
        <li class="inline-block">
          <a class="align-middle link-primary mr-2 mr-lg-0 ml-lg-2" href="/hellenthal-group/publications/">
            publications
          </a>
        </li>
        
      
        
        <li class="inline-block">
          <a class="align-middle link-primary mr-2 mr-lg-0 ml-lg-2" href="/hellenthal-group/research/">
            research
          </a>
        </li>
        
      
        
        <li class="inline-block">
          <a class="align-middle link-primary mr-2 mr-lg-0 ml-lg-2" href="/hellenthal-group/software/">
            software
          </a>
        </li>
        
      
        
      
        
      
    </ul>
  </div>
</header>


    <div>
      <article class="container mx-auto px-2 mt2 mb4">
  <header>
    <h1 class="h0 py-4 mt-3">research</h1>
  </header>
  <div class="col-4 sm-width-full border-top-thin">
  </div>
  <div class="prose mb-4 py-4">
    <font size="5">
	<b>
		Our group develops and applies statistical methodology to apply to large-scale data cohorts to learn about several features:
	</b>
</font>

<hr width="100%"> 

<h3>
	<b>
		<u>
			Describing the genetic structure of worldwide human populations
		</u>
	</b>
</h3>

This includes exploring the fine-scale structure of the <a href="https://www.nature.com/articles/nature14230">United Kingdom</a>, <a href="https://www.nature.com/articles/s41467-018-07748-z">Latin America</a> and elsewhere (<a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007152">Ireland</a>, <a href="https://advances.sciencemag.org/content/5/9/eaaw3492?utm_campaign=toc_advances_2019-09-06&et_rid=17774313&et_cid=2977176">Italy</a>, <a href="https://pubmed.ncbi.nlm.nih.gov/28983069/">Finland</a>), with current work focusing particularly on hundreds of <a href="https://www.biorxiv.org/content/10.1101/756536v1">African ethnic groups</a>. One aim of our studies is to describe how genome-wide genetic structure correlates with such features as geography, shared ethnicity, language, culture and history. We can then unearth the primary factors driving genetic diversity (e.g. the <a href="https://www.pnas.org/content/116/2/593">consolidation of the Kuba Kingdom in the Kasai Region of the Democratic Republic of the Congo</a>) and facilitating the design of future large-scale genetic studies within these regions. Analogous to genetic ancestry testing companies like 23andMe and Ancestry, we use curated genetic data from individuals carefully sampled using birthplace and parent/grandparent information to infer the ancestry of individuals with unknown ancestry, including exploring the implications for e.g. correcting for population stratification in genome-wide association studies (GWAS). 

<br><br>


<figure>
	<img src="/hellenthal-group/assets/img/fineSTRUCTURE-pobi_Lesie2015.png" height="400"/>
	<figcaption>Inferred POBI clusters from Leslie et al (2015) </figcaption>
</figure>

<hr width="100%"> 

<br>

<u>
	<b>
		<h3>
			Extracting the ancestral history of different world-wide human groups
		</h3>
	</b>
</u>

We are also interested in how different groups relate genetically to one another and inferring precisely when different groups intermixed. Our work (with collaborators Simon Myers, Daniel Falush and Dan Lawson) has demonstrated how the genetic make-up of worldwide groups has been influenced by empires and large-scale migrations, e.g. related to the Mongol Empire of Genghis Khan, the Arab Slave Trade (<a href="https://science.sciencemag.org/content/343/6172/747.long">Hellenthal et al 2014</a>), Norse Viking Anglo-Saxon invasions of the United Kingdom (<a href="https://www.nature.com/articles/nature14230">Leslie et al 2015</a>), <a href="https://www.nature.com/articles/s41467-018-07748-z">the clandestine colonial-era migrations of Sephardic Jews to the Americas</a>, <a href="https://www.cell.com/ajhg/fulltext/S0002-9297(17)30291-4"> and the migration of Zoroastrians to India</a>. We have also unearthed mixing events with little recorded evidence, such as European-like ancestry in isolated groups within Pakistan and China (Hellenthal et al 2018). With collaborators Joachim Burger, Daniel Wegmann, Krishna Veeramah and others, we have also incorporated DNA from ancient human remains to learn about, among other things, <a href="https://science.sciencemag.org/content/353/6298/499.abstract">the migrations of genetically diverged communities among the world's first farmers</a> and the genetic origins of peoples with <a href="https://www.pnas.org/content/115/13/3494">elongated skulls in Medieval Bavaria</a> . A webpage summarizing the dates and proportions of intermixing events inferred in >70 world-wide groups is <a href="www.admixturemap.paintmychromosomes.com">here</a>.


<br>
<br>

<figure>
	<img src="/hellenthal-group/assets/img/ChromopainterSharingMap-Broushaki.jpg" width="540"/>
	<figcaption>Plot of haplotype-sharing differences between two ancient individuals from Broushaki et al (2016) </figcaption>
</figure>

<hr width="100%"> 

<br>

<u>
	<h3>
		<b>
			Identifying genetic loci that have facilitated human's adaptation to different geographic regions and disease pressures
		</b>
	</h3>
</u>


<br>


This includes new techniques that account for admixture and demography when identifying the specific genetic regions and populations that show signs of such adaptation (i.e. selection), allowing us to narrow in on the time periods over which selection has acted.

<br/><br/>


<figure>
	<img src="/hellenthal-group/assets/img/BetaBinomFigNODONHaakColors.jpg" height="300"/>
</figure>

<hr width="100%"> 


<u>	
	<h3>
		<b>
			Identifying differentially methylated sites (DMS) among populations 
		</b>
	</h3>
</u>

DMS may indicate how specific environments may turn on or off certain genetic pathways. We aim to combine information across multiple datasets and populations to increase precision to find DMS related to tissue and/or ethnic differences. Along with collaborators Matt Silver, Andrew Prentice and Robert Waterland, we have a particular focus on metastable epialleles that have been shown to be particularly susceptible to environmental influences experienced by embryos in utero. 

<br/><br/>


<figure>
	<img src="/hellenthal-group/assets/img/Methylation_fig.png" width="540"/>
	<figcaption>Strategy for identifying correlated regions of systemic interindividual variation from Gunasekara et al (2019) </figcaption>
</figure>

<hr width="100%"> 

  </div>
</article>

    </div>

    <div class="border-top-thin clearfix mt-2 mt-lg-4">
  <div class="container mx-auto px-2">
    <p class="col-8 sm-width-full left py-2 mb-0">This project is maintained by <a class="text-accent" href="https://github.com/sahwa">sahwa</a></p>
    <ul class="list-reset right clearfix sm-width-full py-2 mb-2 mb-lg-0">
      <li class="inline-block mr-1">
        <a href="https://twitter.com/share" class="twitter-share-button" data-hashtags="hellenthal-group">Tweet</a> <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document, 'script', 'twitter-wjs');</script>
      </li>
      <li class="inline-block">
        <a class="github-button" href="https://github.com/sahwa/" data-icon="octicon-star" data-count-href="sahwa//stargazers" data-count-api="/repos/sahwa/#stargazers_count" data-count-aria-label="# stargazers on GitHub" aria-label="Star sahwa/ on GitHub">Star</a>
      </li>
    </ul>
  </div>
</div>

  </body>

</html>
